src/openzaak/components/documenten/tests/test_upload_cmis.py

```diff
@@ -286,13 +288,17 @@ def test_update_eio_metadata(self):
         self.assertEqual(eio_new.versie, 2)
         self.assertEqual(eio_new.titel, "another summary")
         self.assertEqual(eio_new.bestandsomvang, 4)
-        self.assertEqual(eio_new.canonical.bestandsdelen.count(), 0)
+        self.assertEqual(
+            BestandsDeel.objects.filter(informatieobject_uuid=eio_new.uuid).count(), 0
```

Comment:
I am wondering why this has changed. Will look when Django 4.2 is merged.

Reply:
Hi! This change was made in https://github.com/open-zaak/open-zaak/issues/1597 
This Pr is only to run `pyupgrade` and `django-upgrade` and remove soon to be deprecated syntax

Explanation: In Django 4.2 the descriptor for reverse-fk fields was change. In the function `create_reverse_many_to_one_manager` in the `get_queryset` they added a check:
```
  if self.instance.pk is None:
      raise ValueError(
          f"{self.instance.__class__.__name__!r} instance needs to have a "
          f"primary key value before this relationship can be used."
      )
```
This check breaks all attempts to get reverse-fk fields for instances without `pk`. In this case `eio_new.canonical` doesn't have a pk, so it crushes now.